yuptoo/lib/logger.py

```diff
@@ -34,21 +34,21 @@ def initialize_logging():
     if os.environ.get("CLOWDER_ENABLED", "").lower() == "true":
         aws_access_key_id, aws_secret_access_key, aws_region_name, aws_log_group = cloudwatch_config_values()
         if all((aws_access_key_id, aws_secret_access_key, aws_region_name, aws_log_group)):
-            from boto3.session import Session
+            import boto3
             import watchtower
 
-            boto3_session = Session(
+            boto3_client = boto3.client(
+                'logs',
                 aws_access_key_id=aws_access_key_id,
                 aws_secret_access_key=aws_secret_access_key,
                 region_name=aws_region_name
             )
 
             # configure logging to use watchtower
             cw_handler = watchtower.CloudWatchLogHandler(
-                boto3_session=boto3_session,
+                boto3_client=boto3_client,
                 log_group=aws_log_group,
-                stream_name=socket.gethostname(),
-                create_log_group=False
+                stream_name=socket.gethostname()
```

Comment:
Could you share any example value of `socket.gethostname()` ?
OR can we simply hard-code this value to something like `yuptoo`?

Reply:
Here is an example from puptoo - [https://github.com/RedHatInsights/insights-puptoo/blob/1ca2806b2ce007c48538b90ab3b786b921fdef35/src/puptoo/utils/puptoo_logging.py#L69](https://github.com/RedHatInsights/insights-puptoo/blob/1ca2806b2ce007c48538b90ab3b786b921fdef35/src/puptoo/utils/puptoo_logging.py#L69)

Reply:
I think by adding socket.gethostname() we can differentiate between individual pod logs, as yuptoo can scale horizontal it can have multiple pods running. Though not sure in which scenario it will help.
Example of puptoo log stream in kibana - `@log_stream:puptoo-processor-8754f6586-88ndn` log_stream takes the pod name in our case it will be `yuptoo-****`